src/includes/shape.js

```diff
@@ -422,6 +422,12 @@ pub.rect = function(x, y, w, h, tl, tr, br, bl) {
   if(arguments.length > 4) {
```

Comment:
How about instead placing it after *this* line (so if using one radius, it will also be checked)? Also, what about this syntax:

```js
if( pub.min(Array.prototype.slice.call(arguments, 4)) < 0 ) {
  error("b.rect(), needs positive values as arguments for the rounded corners.");
}
```

Reply:
Moved it up. I have no feelings about how to write it. What's the difference? Is it faster?

Reply:
No, I thought maybe mine is a bit shorter. Yours is more legible though, so just keep it. The only thing is, that I would get rid of the `return` statement. We generally don't use return after throwing errors.

Reply:
Old habit. Get of my lawn you damn kids!